### ISSUE TITLE
Adjust font size for description headings

### ISSUE THREAD

**Issue by trugwaldsaenger** [hbz/oerworldmap#967]
When using markdown heading h1 the fontsize becomes huge. In order to avoid editors using h2 (or even h3) when editing headings, whe should adjust this.

Example:
https://oerworldmap.org/resource/urn:uuid:ab0c598e-0d1e-4920-a361-6f7dca99db25

**Comment by acka47** [comment]
I think it is ok the way it is. The profile title already is in h1 which becomes obvious when you also use h1 in the body. So most people will switch to h2 in the which is ok, see the updated resource.

**Comment by literarymachine** [comment]
> I think it is ok the way it is. The profile title already is in h1 which becomes obvious when you also use h1 in the body. So most people will switch to h2 in the which is ok, see the updated resource.

I agree. We should maybe add this to the editing guidelines. There should not be more than one h1, which is the profile / story title at the very top.

**Comment by trugwaldsaenger** [comment]
If this is OK for you guys, it will be OK for me as well! @karindr: Could you add this to our FAQ for editors?

**Comment by j0hj0h** [comment]
I agree that highest headline level in the content should be h2 and h1 reserved for the title. I see the following possibilities to achieve this:

- Highest level entered is just h2
  - because editors read the corresponding guideline
  - because we introduce some kind of validation, that doesn't allow h1
- We shift the markdown headline levels during rendering

I'd vote for validation or shifting.

**Comment by formimkontext** [comment]
I agree and would vote for validation or shifing, too.

**Comment by karindr** [comment]
I also agree, validation and shifting is safer than rely on people reading the FAQ. 

Please keep in mind to change the other headings (h2, h3, h4 and so an) too and also for html input.

Should we put it to the FAQ nevertheless or should we keep this list short, @trugwaldsaenger?

**Comment by j0hj0h** [comment]
Validation AND shifting doesn't make sense. If we validate, there is nothing to shift anymore and other way round. If we validate we should also put it to the FAQ, if we shift, there is no need to inform about.

**Comment by literarymachine** [comment]
Honestly, I believe this is absolute overkill. I would opt for a simple short guideline above the input. Keep in mind that you originally wanted to give people full flexibility with in #882...
